### PR TITLE
Lintr suggestions

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

**Comment by PavelBal** [comment]
While playing around with Visual Studio Code, I used `lintr` and implemented some of the suggestions. I think it is quite useful as at one place it revealed a typo in variable name, and at other place pointed to a not used variable.

File: R/project-configuration.R [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #' @description An object storing configuration used project-wide
 #' @format NULL
 #' @export
-ProjectConfiguration <- R6::R6Class(
+ProjectConfiguration <- R6::R6Class( # nolint: object_name_linter.
```
Comment (PavelBal) [inline R/project-configuration.R:6]: Bit annoying that it complains about our class names. Maybe there is an option to define a separate style for R6 class names?

File: R/sensitivity-calculation.R [hunk 2]
```diff
@@ -208,17 +211,17 @@ sensitivityCalculation <- function(simulation,
       warning(messages$noPKDataToWrite())
     } else {
       # Convert tidy data to wide format for each output path
-      pkData_wide_list <- purrr::map(
+      pkDataWideList <- purrr::map(
```
Comment (PavelBal) [inline R/sensitivity-calculation.R:214]: Renamed the variable.

File: R/project-configuration.R (hunk 1 above)
Comment (Felixmil) [reply R/project-configuration.R:6]: You can use these settings I setup for test in TLF package: https://github.com/Felixmil/TLF-Library/blob/gh-actions-test/.lintr

File: .lintr [hunk 3]
```diff
@@ -0,0 +1,4 @@
+linters: linters_with_defaults(
```
[the remark this answers is not in the document]
Comment (Felixmil) [reply .lintr:1]: Yes it is ! Lintr rules should be harmonized by project/company, not user so it makes perfect sens.

File: R/project-configuration.R (hunk 1 above)
Comment (PavelBal) [reply R/project-configuration.R:6]: nice

**Comment by PavelBal** [comment]
@Felixmil it's up to you if you want to add anything else to the default linter config. I would prefer to keep `object_length_linter` and `cyclocomp_linter` as the rules do make sense and should be reminders for us not to make the code too complex.

**Comment by Felixmil** [comment]
> @Felixmil it's up to you if you want to add anything else to the default linter config. I would prefer to keep `object_length_linter` and `cyclocomp_linter` as the rules do make sense and should be reminders for us not to make the code too complex.

Should be enough for now, I think better code test coverage is more important (reach >90%).

File: R/utilities-population.R [hunk 4]
```diff
@@ -57,7 +57,7 @@ readPopulationCharacteristicsFromXLS <- function(XLSpath, populationName, sheet
 #' Possible gender entries as integer values
 #'
 #' @export
-GenderInt <- enum(list(
+GenderInt <- enum(list( # nolint: object_name_linter.
```
Comment (Felixmil) [inline R/utilities-population.R:60]:
```suggestion
GenderInt <- enum(list(
```

File: R/utilities-population.R [hunk 5]
```diff
@@ -164,7 +164,7 @@ extendPopulationFromXLS <- function(population, XLSpath, sheet = NULL) {
 
 #' Supported distributions for sampling
 #' @export
-Distributions <- enum(list(
+Distributions <- enum(list( # nolint: object_name_linter.
```
Comment (Felixmil) [inline R/utilities-population.R:167]:
```suggestion
Distributions <- enum(list(
```

**Review by Felixmil — APPROVED** [review]

**Pull request merged**